xml/obs_ag_installation_and_configuration.xml

```diff
@@ -249,6 +249,47 @@ systemctl start obswarden.service</screen>
      not do this on a system connected to an untrusted network or make sure to
      block the ports with a firewall.</para>
    </warning>
+   <sect3 xml:id="_cloud_upload_setup">
+    <title>Cloud Upload Setup</title>
+    <para>In order to setup the Cloud Upload feature you will need to configure the tools required per each cloud provider.
+      Right now we only support the AWS Amazon Cloud (<link xlink:href="https://aws.amazon.com"/>) as provider.
+    </para>
+    <sect4 xml:id="_aws_authentication_workflow">
+      <title>How does the AWS's authentication work for uploading images?</title>
+      <para>
+        We are going to use the role based authentication provided by Amazon to enable the OBS instance to upload images to other user's accounts.
+      </para>
+      <para>
+        The users will obtain an external ID (automatically created and unique) and the OBS account ID to create an Identity and Access Management (IAM) role.
+        After the user created the role, he needs to provide the Amazon Resource Name (ARN) of the role to OBS which stores it in Cloud::EC2::Configuration.
```

Comment:
Sorry for the late review and it is already merged ...

> which stores it in Cloud::EC2::Configuration

I would not include that as it is developer documentation. For the admin it is not relevant in which model / database table this information get stored. I would rather include that the user can set it on ``/cloud/configuration/ec2`` for instance when you want to include that.

Maybe I would also include a sentence that therefore an uploader account is necessary which gets configured in the next step (see AWS authentication credentials setup)